.github/workflows/default.yml

```diff
@@ -190,6 +190,10 @@ jobs:
       with:
         named-caches-hash: ${{ hashFiles('python*.lock', 'tools/*.lock') }}
         cache-lmdb-store: 'true'
+    - name: Run halfstack etcd container
+      run: |
+        sed -i'' "s/- \"8120:2379\"/- \"8121:2379\"/g" docker-compose.halfstack-main.yml
+        docker-compose -f docker-compose.halfstack-main.yml run -d --service-ports backendai-half-etcd
```

Comment:
This should be removed and replaced with the `ai.backend.testutils.bootstrap.etcd_container` fixture to keep it parallelization-safe when running `pants test`. Please refer other use cases.